src/panels/lovelace/cards/types.ts

```diff
@@ -97,6 +97,9 @@ export interface ConfigEntity extends EntityConfig {
 
 export interface PictureGlanceEntityConfig extends ConfigEntity {
   show_state?: boolean;
+  attribute?: string;
+  prefix?: string;
+  suffix?: string;
```

Comment:
Remove this, you can add `unit` instead of suffix maybe.

Reply:
Oh crap, we already have `prefix` and `suffix` in other places 😕

Reply:
I don't mind the `suffix` keyword but I think we should align them all, right? Gauge and Sensor card use Unit.

Reply:
SHIP IT